### DISCUSSION TITLE
Kartenbild herunterladen

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#66] [Q&A]
Ich habe am 14 Zoll Laptop versucht ein Kartenbild über "aktuellen Ausschnitt..." herunterzuladen.
Die Ansicht im Browser habe ich aus Darstellungsgründen auf 80% gezoomt. Die Bildgrösse im Webgis5 wurde automatisch auf 1479 x 945 angepasst. 
siehe screenshot
![image](https://user-images.githubusercontent.com/124671123/235685778-b0afcc5f-f390-451c-99c8-9dc8d191d112.png)

Wenn ich dann auf Kartenbild herunterladen gehe kommt folgende Meldung.
 
![image](https://user-images.githubusercontent.com/124671123/235701559-85bf9878-2afe-42ee-b7d0-7e16a4e04df9.png)

Ich gehe dann davon aus, dass ich die Bildgrösse im WebGIS anpassen soll. Die lässt sich aber nicht anpassen, weil die Felder nicht editierbar sind. Somit funktioniert auch das herunterladen nicht.

Ein Anwender der nicht checkt, dass er die Ansicht im Browser verändert hat, also auf 80, 90 oder wieviel % auch immer gezoomt hat, wird wahrscheinlich kaum darauf kommen, dass das herunterladen des Kartenausschnittes deshalb nicht funktioniert.

**Comment by gis-eni** [comment]
Der Wunsch, dass der Anwender die Größe nicht selber einstellen kann kommt aus einem früheren Post im Forum.
Lösung: Anpassung der Fehlermeldung:

![image](https://user-images.githubusercontent.com/62870568/235845048-49ca55b6-b2d9-48af-9e93-ca9e328da15d.png)
